app/controllers/groups_controller.rb

```diff
@@ -36,13 +40,28 @@ def update
       flash[:notice] = "Group '#{@group.name}' updated!"
       redirect_to group_path(@group)
     else
-      flash[:warning] = 'An error occurred, please try again.'
+      flash[:warning] = 'Please enter valid information.'
       redirect_to edit_group_path(@group)
     end
   end
 
   def destroy
-    # Destroy not working. Delete lists associated with group first.
+    @lists = List.where("group_id = :group_id", {group_id: @group.id})
```

Comment:
showing you alternative: `List.where(group_id: @group.id)`

Reply:
Much simpler, I'll use it.